Quantum-Control-Applications/ESR/Use case 1 - Sekhar Lab - CPMG/README.md

```diff
@@ -0,0 +1,102 @@
+train of pulses is to isolate the NV ensemble from noise at certain frequencies. Thus, it can
+serve to extend the lifetime of information encoded in qubits. A CPMG sequence can also be used to
+probe the noise environment surrounding the spins [1]. The image below displays the
+train of pulses applied in this experiment. After the hyperpolarization of spins, an initial
+`X_pi/2` pulse is sent and it is followed by a train of `Y_pi`. The delay between two consecutive
+`Y_pi` pulses is `2*tau`. In ESR experiments the echo amplitude is detected at `tau` time after
+`Y_pi` pulses.
+
+![sequence](CPMG_seq.png)
+
+### Echo amplitude decay in CPMG
+
+The measured decay traces with CPMG are shown below. The `y-axis` list the echo amplitude as demodulated
+and averaged in the OPX as the sequence evolves for `10 ms`. The `x-axis` is the `Time of echo` which
+is the time from the beginning of the sequence at which an *echo* was measured. The legend list all
+the different values of $\tau$ that were used. The traces displayed were obtained with only ten
+averages and in the timespan of a few minutes. The data transferred from the OPX to the computer was
+the data visualized in the figure and no post-processing was needed.
+
+We fit the experimental data with a double exponential following the equation below. For the rest of
+the analysis we focus on the measured $T_{2}$. A discussion about the `CPMG tail` can be found in
+Ref. [2]. The relative contribution of $T_{2}$ is at least three times larger than the `CPMG tail`.
+
+$$
+\begin{align}
+Echo\;amplitude &= A\exp(-t/T_{2}) + B\exp(-t/T_{tail})
+\end{align}
+$$
+
+
+![decays](cpmg_iq_decays_1.png)
+
+The table below list the key parameters extracted from the CPMG sequence. $T_{2}$ values are in the 
+order of $\mu$s and are listed in the first column. Note that the $T_{2}$ does not increases
+monotonically as $\tau$ increases. BW is the intrinsic linewidth for a Lorentzian peak and is calculated
+as $1/2\pi T_{2}$. The CPMG sequence acts as a filter of the power spectral density, thus the center 
+frequency can be calculated as $\pi/\tau$. The relative amplitude of the contribution
+of $T_{2}$ with respect to the tail are listed in the last column.
+
+| τ [μs]      | T2 [μs]      | BW (1/2πT2) [kHz] | Center frequency (π/τ) [kHz]   | T2 relative amplitude A/(A+B) [%] |
+| :---:        | :---:        |    :----:   |          :---: |     :---: |
+| 2.00 | 20.33     | 7.83       | 1570.80   | 78.69 |
+| 3.09 | 97.58   | 1.63        | 1017.36      | 69.11 |
+| 4.77 | 15.66     | 10.16       | 658.34  | 88.71 |
+| 7.37 | 163.12   | 0.98        | 426.38      | 88.46 |
+| 11.38 | 52.09    | 3.06       | 276.06   | 99.00 |
+| 17.58 | 99.01   | 1.61        | 178.74      | 96.98 |
+| 27.14 | 63.04     | 2.52       | 115.74  | 92.26 |
+| 41.92 | 100.00   | 1.59        | 74.94      | 98.48 |
+| 64.75 | 206.68     | 0.77       | 48.52   | 94.18 |
+| 100.00| 209.26   | 0.76        | 31.42      | 95.05 |
```

Comment:
From the figure it seems that T2 decreases when tau increases, from the data it seems otherwise...
But maybe in the figure we don't have all of the lines?

Reply:
Ok, in the figure we don't have all of the lines, so it's a bit confusing (longest T2 seems to be tau=7us).